Mu2eG4/geom/extmon_fnal_v02.txt

```diff
@@ -179,12 +179,11 @@ bool extMonFNAL.stackMotherSolid = true;
 // Scintillator parameters
 
 vector<double> extMonFNAL.scintHalfSize = { 54.0 , 42.0, 3.0 };
-string         extMonFNAL.scintMaterial = "G4_C";
-double         extMonFNAL.scintOffset = 29.0;
-double         extMonFNAL.scintInnerOffset = 33;
-double         extMonFNAL.scintGap = 5.3;
+string         extMonFNAL.scintMaterial = "Scintillator";
+double         extMonFNAL.scintOffset = 29.0; //Distance from pixel plane center to center of scintillator on outter ends of detector table
```

Comment:
Do you mean the scintillator closest to the pixels?

Reply:
Yes I do, in particular the scintillator closest to the pixels that is attached to the double mounted scintillator setup.